README.md

```diff
@@ -59,4 +59,14 @@ nfcore/rnaseq is now used by a number of core sequencing and bioinformatics faci
     <td>Quantitative Biology Center (QBiC), Germany</td>
     <td>https://portal.qbic.uni-tuebingen.de/portal/</td>
   </tr>
+  <tr>
+    <td><img src="docs/images/gis_logo.png" width="100"></td>
```

Comment:
This is fine for now, but just so you know I was planning on removing this section from all pipelines and instead just having a single page on http://nf-co.re (I started a section on the homepage, but the link doesn't go anywhere yet).